uncertainty_toolbox/utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+
+def check_flat_same_shape(*args):
+
+    first_shape = args[0].shape
+    for arr in args:
+        try:
```

Comment:
Why does this need to be try-except? It seems like a if-then would suffice. Also in general when doing try and except you should try to only wrap the code in the try blocks with things that could fail. Are all lines prone to failing or just one? Also could add more detailed except catching if this really is needed.

Reply:
I actually changed this during our meeting.